depends/windowsstore/libxml2/02-fix-uwp-api.patch

```diff
@@ -1,3 +1,21 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 913da3e..07cbec1 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -147,7 +147,12 @@ target_include_directories(
+ target_compile_definitions(libxml2 
+   PRIVATE 
+   _CRT_SECURE_NO_WARNINGS
+-  _CRT_STDC_NO_DEPRECATE)
++  _CRT_SECURE_NO_DEPRECATE
++  _CRT_STDC_NO_DEPRECATE
++  _CRT_NONSTDC_NO_DEPRECATE
++  HAVE_WIN32_THREADS
++  _WINSOCK_DEPRECATED_NO_WARNINGS
++)
+ 
+ if(BUILD_SHARED_LIBS AND MSVC)
+   target_link_libraries(libxml2 PRIVATE ws2_32.lib)
```

Comment:
@afedchin applying this hunk of the patch fails. I've no idea where you got that from.

Reply:
It worked well 3 months ago. What was changed?

Reply:
No, it never applied. Before https://github.com/xbmc/xbmc/pull/15321 failing patches were just ignored.